gazebo_plugins/worlds/gazebo_ros_ackermann_drive_demo.world

```diff
@@ -0,0 +1,660 @@
+<?xml version="1.0"?>
+<!--
+  Gazebo ROS ackermann drive plugin demo
+
+  Try sending commands:
+
+    ros2 topic pub /demo/cmd_demo geometry_msgs/Twist '{linear: {x: 5.0}}' -1
+
+    ros2 topic pub /demo/cmd_demo geometry_msgs/Twist '{angular: {z: 0.1}}' -1
```

Comment:
Since Ackermann steering can't rotate in-place, this command doesn't do anything. We need to add a linear component for it to move.